index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.2.0/css/all.css"
+      integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="CSS/styles.css" />
+    <link rel="stylesheet" href="CSS/main.css" />
+    <title>Natnael-Portfolio</title>
+  </head>
+  <body>
+    <section class="container">
+      <nav class="normal-button">
+        <div class="logo">
+          <div class="placeholder">
+            <div><a href="/"></a></div>
+            <div>N-A</div>
+          </div>
+        </div>
+
+        <button type="button" class="hamburger" id="hamburger">
+          <i class="fas fa-bars"></i>
+        </button>
+        <button type="button" id="close" >
+          <i class="fas fa-times"></i>
+        </button>
+        <ul class="nav-ul">
+          <li><a href="#portfolio" class="nav-text-1">Portfolio</a></li>
+          <li><a href="#about" class="nav-text-2">About</a></li>
+          <li><a href="#contact" class="nav-text-3">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] On your mobile menu, the links except `#portfolio` work when I click them. The reason the portfolio link doesn't work is that you don't have a portfolio section. 

Please what happened to your portfolio section? Kindly fix it.